### ISSUE TITLE
Fastq expected format?

### ISSUE THREAD

**Issue by wsphillips** [alexdobin/STAR#617]
Hi,

I have fastq files from a STRT-seq-2i experiment that are "ordered" but I think something about the format is throwing things off, resulting in paired sequences getting matched up incorrectly and throwing the "Fastq sequences are out of order" error. 

screenshot: https://www.dropbox.com/s/b5lauq3zr1vlp0a/fastqOutOfOrder.png?dl=0

Sequences ordered C1,C2,C3...etc. My guess is its a fastq formatting issue? According to FASTQC the data is in Illumina 1.5, sequenced on a HiSeq2500. 

Recommendations?

**Comment by alexdobin** [comment]
Hi Wiktor,

the mate1/2 files have to be ordered the same way for proper processing.
Illumina pipeline should produce correctly ordered files, I would recommend that you check that it completed without errors.

Cheers
Alex

**Comment by wsphillips** [comment]
For posterity, there were two things I did to resolve this: 

1) Illumina 1.5+ (pre 1.8+, **but critically different from 1.3+**) has an odd 64 character phred scoring system that is offset unlike the others (see the Fastq wiki page). I used https://bioinf.shenwei.me/seqkit/ to convert everything from Illumina 1.5+ to standard sanger format. 

2) Also, my reads were ordered correctly but the naming was different between files (e.g. xxxx_read1_seqnrXXX + xxxx_read2_seqnrXXX) I used seqkit again to homogenize the names while preserving the sequence numbers and all seems to work correctly now.

Since solved I will close the issue, but hopefully this info can help others if they run into similar problems.
